sources/core/Stride.Core.DiagnosticsServices.Tests/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Is this file needed if you're including `using Xunit` everywhere?

Reply:
ive changed it to use csproj usings and removed it everywhere in the cleanup